agent/containers/images/visualizers/grafana_setup.py

```diff
@@ -0,0 +1,99 @@
+"""
+Sets up all grafana plugins, data sources, and dashboards through grafana API.
+Chooses what to upload/enable based off VIS_TYPE environment variable.
+
+If VIS_TYPE is 'live' (default for live-metric-visualizer):
+    - Node exporter/DCGM dashboards will be uploaded
+    - Prometheus data source will be configured
+    - Grafana-pcp plugin will be enabled
+    - PCP redis and vector datasources will be configured
+    - PCP default dashboards will be enabled
+If VIS_TYPE is 'prom' (default for prom-graf-visualizer):
+    - Node exporter/DCGM dashboards will be uploaded
```

Comment:
You say "will be uploaded", while in the other two lists "PCP default dashboards" are "enabled". I'm assuming this means that the PCP dashboards are indeed present but not enabled by default whereas node exporter and dcgm have to be added; however, (unless that doesn't actually "enable" them) it might be better to say "uploaded and enabled" so there's no doubt.